### DISCUSSION TITLE
Noob question :: What specs a server (live) has to meet in order to use Statamic

### DISCUSSION THREAD

**Discussion by eduard-un** [statamic/cms#6152] [Q&A]
Up to this point, everything is pretty clear on how to build my layouts while working locally. 

The question I have is whether there is any documentation dealing with taking a local install and putting it live on shared hosting. I'm referring to my server, where I have a hosting package already purchased. I can have WordPress installed without any issues, but I could not find any documentation on how to deal with the Statamic website in such a case.

If anyone has any links to documentation related to this topic, it would be very much appreciated. 

If it matters, I have a shared hosting package that includes the followings: 

- Space 15000 MB (15 GB)  SSD
- Free  SSL certificate included
- IP address, dedicated website, and mail
- Daily Backup  Files and MySQL
- MultiPHP Selector 5.4 to PHP 7.3
- Server MySQL 5.7 si MariaDB 10.2
- NGINX High-Performance Caching

Do I need to have `Git project repository & composer`?

Thank you in advance.

**Comment by edalzell** [comment]
Shared hosting is not recommended, you're better off using a VPS provider like Digital Ocean. You can see a list of recommended hosts here: https://statamic.dev/requirements#recommended-hosts

**Reply by jayrdi** [reply]
I am building a site for the small company I work for and VPS will either be too expensive or require too much server management from me to be viable. Does "not recommended" mean that it's going to be prohibitively difficult to get running and also cause issues down the line? I also like the offered mail servers that those web hosting services provide. I wouldn't want to have to set up my own mail server on a VPS. Is it actually possible to run Statamic on shared hosting?

**Reply by edalzell** [reply]
It means you'll have to fight it a bit to do "normal" things.

VPSes are not expensive these days: https://www.vultr.com/pricing/ and ploi.io has a free tier for one server. But Forge is only like $9/month or something, well worth it to cover your time that you'd spend fighting w/ shared hosting.

**Comment by jayrdi** [comment]
Hi @eduard-ungureanu, I'm approaching the same question - did you have any luck getting set up?

**Reply by edalzell** [reply]
Where are you trying to deploy to?

**Reply by jayrdi** [reply]
I wanted to try Krystal that was recommended to me as a good UK provider:

https://krystal.uk/web-hosting/web-hosting-features

**Reply by edalzell** [reply]
Looks like the Ruby package allows SSH access, so that might be not too bad.

**Comment by edalzell** [comment]
> VPS will either be too expensive or require too much server management from me to be viable

Not really sure what this means? most sites run fine on a $5 DO droplet and Ploi has a free tier. Once it's set up it's fairly easy to manage.

**Reply by jayrdi** [reply]
Okay, maybe I'll try DO or something. My limited experience was that VPS got really expensive fast. I also have The Fear that I'll end up spending a lot of my time managing and maintaining the server, with only limited experience and no support..

If it's reasonably straightforward though, I'll give it a go... Not sure what to do about email provision (needs to use the hostnames) cos I heard that was very difficult on a VPS?

Thanks for your advice!

**Reply by edalzell** [reply]
> Not sure what to do about email provision

I recommend Fastmail.

**Reply by jayrdi** [reply]
Okay, I'll give that a look, thanks!
